### DISCUSSION TITLE
Documentation improvement (TEKSI User DAY)

### DISCUSSION THREAD

**Discussion by ponceta** [QGEP/QGEP#819] [Ideas]
The actual TEKSI module documentation is not usable for end users because of the following : 

 - La documentation n'est pas traduite
 - La traduction n'est pas complète
 - La documentation utilisateur est mélangée aux autres documentations
 - La navigation dans la documentation n'est pas facile
 - La recherche devrait pouvoir être effectuée sans risque de sortir de la documentation utilisateur
 - La documentation métier n'est pas présente
 - Les métiers en présence on besoin d'interface et donc de documentations spécifiques
 1. Géomaticien
 2. Exploitant Réseau
 3. Planificateur PGEE

 - Les utilisateurs parlent français ou allemand et sont donc capables de documenter dans cette langue avant de pouvoir soumettre leurs inputs pour une éventuelle traduction
 - L'outil de documentation actuel nécessite des compétences informatiques pour écrire la documentation (à l'inverse d'une documentation word ou autre logiciel maitrisé par les utilisateurs)
 - La documentation doit pouvoir s'appuyer sur des normes professionnelles ou des supports officiels disponibles en ligne (glossaire / WIKI VSA, ...)

Idées
 - Selon le profil utilisateur, la documentation est adaptée

Challenges
- Contributions venant de plusieurs languages

Bonnes pratiques :
- Ne pas intégrer de fichiers ou codes spécifiques dans la documentation (priviliégier les liens vers les ressources)

Contributions :
@kandre @rbovard @ponceta Lukas Wiss, Fabien Cattin

**Comment by sjib** [comment]
The VSA has established the "[Wegleitung Daten der Siedlungsentwässerung](https://vsa.ch/Mediathek/gep-datachecker-jahresgebuehr/)" / "[Guide des données sur l’assainissement urbain](https://vsa.ch/fr/M%C3%A9diath%C3%A8que/plateforme-wiki-vsa-dss-datachecker-pgee-guide-licence-individuelle/)" that covers a lot of the requested topics, mainly how to model reality object like sewers and manholes into the datamodel VSA-DSS (which is the basis of TEKSI). It is published on the so called Wiki-Plattform www.vsa.ch/wiki an includes also:
German content list:
- Wegleitung Daten der Siedlungsentwässerung (ehemals Wegleitung GEP-Daten)
-  GEP-Datachecker
-  Datenstruktur Siedlungsentwässerung ([VSA-DSS](https://vsa.ch/glossar/vsa-dss/))
-  Begleitdokumente zu den Datenmodellen 2020
-  Glossar mit Modellbegriffen
-  Zugriff auf Neuerungen

French content list:
-  Guide des données sur l’assainissement urbain (anciennement Guide des données PGEE)
-  Vérificateur de données PGEE (Datachecker PGEE)
-  Structure de données de l’assainissement urbain (VSA-DSS)
-  Documents d’accompagnement pour les modèles de données 2020
-  Glossaire des termes relatifs aux modèles
-  Accès aux nouvelles fonctionnalités

All topics with a lot of specific examples (e.g. see below)

So to not re-invent the wheel I suggest to use this Wiki plattform as a integrated information source for the updated and improved TEKSI documentation and link existing content where already available and well presented.

![vsa_wiki_directives_de_saisie_overview](https://github.com/QGEP/QGEP/assets/1265865/d378b858-450b-4870-84f6-ac5f7942fea6)

![vsa_wiki_directives_de_saisie_exmaple_OAS_OAP](https://github.com/QGEP/QGEP/assets/1265865/7a87d69e-affc-4e82-8d17-976039dc27c7)


@kandre @rbovard @ponceta Lukas Wiss (and @urskaufmann), Fabien Cattin Please have a look at the https://vsa.ch/fr/wiki_categories/3-exemples-de-saisie/ resp. https://vsa.ch/wiki_categories/erfassungsbeispiele/ and make a list here of what would be missing in detail.

Then either it can be integrated in the VSA Wiki or first be developed on TEKSI side.

**Comment by urskaufmann** [comment]
Integrate to vsa-examples: basically good idea. If TEKSI-members have an account for vsa-wiki. We talked about that in the usergroup. VSA-fee part of the TEKSI fee. Is this the idea?
Are there permanent-links to this vsa-examples?
The examples show first the dss-mini configuration (that can not be used with TEKSI). These dss-mini-part's should not be viewable, if we link to the wiki (it's just confusing). 
And the descriptions in the vsa-dss-part should be rewriten (it is often to near to the dss-mini description, I mean specially the wastewater-nodes without wastewater-structure). My opinion.
